packages/core/crates/core-types/Cargo.toml

```diff
@@ -1,8 +1,7 @@
 [package]
 name = "core-types"
 version = "0.1.1"
-authors = ["HOPR Association <tech@hoprnet.org>"]
-description = "Contains types required by the Core package, which are not generic enough to be in the utils-types crate"
+authors = ["NumberFour8 <lukas.pohanka@inina.net>"]
```

Comment:
sure you want your private address there?

Reply:
Now immortalized in this comment.

Reply:
```suggestion
authors = ["HOPR Association <tech@hoprnet.org>"]
```